clients/hadoopfs/src/main/java/io/lakefs/LakeFSFileSystem.java

```diff
@@ -402,15 +382,22 @@ private boolean renameObject(LakeFSFileStatus srcStatus, Path dst) throws IOExce
      * @return an IOException that corresponds to the translated API exception
      */
     private IOException translateException(String msg, ApiException e) {
+        IOException ex;
         int code = e.getCode();
         switch (code) {
             case HttpStatus.SC_NOT_FOUND:
-                return (IOException) new FileNotFoundException(msg).initCause(e);
+                ex = new FileNotFoundException(msg);
+                ex.initCause(e);
```

Comment:
Cool!  Can we also _not_ break it up into 2 lines, matching the [documentation on oracle.com](https://docs.oracle.com/javase/7/docs/api/java/lang/Throwable.html#initCause(java.lang.Throwable))?

Reply:
sure - I'll revert the code back and match the exception type